http/src/request/application/interaction/create_followup.rs

```diff
@@ -208,9 +208,10 @@ impl<'a> CreateFollowup<'a> {
 
     /// Set the message's flags.
     ///
-    /// The only supported flag is [`EPHEMERAL`].
+    /// The only supported flags are [`EPHEMERAL`] and [`SUPPRESS_EMBEDS`].
```

Comment:
actually, according to the [current docs], only ephemeral is supported. if suppress embeds is in fact supported, a newer version of documentation will need to be cited.

[current docs]: https://discord.com/developers/docs/interactions/receiving-and-responding#create-followup-message

Reply:
It functions the same as the Execute Webhook endpoint which supports the `SUPPRESS_EMBEDS` flag.

Reply:
Oops! Unclear wording in my link led me to believe that *only* `64` was supported. Apoligies.